iroh-quinn-proto/src/connection/paths.rs

```diff
@@ -107,7 +107,7 @@ pub struct RttEstimator {
 }
 
 impl RttEstimator {
-    fn new(initial_rtt: Duration) -> Self {
+    pub(super) fn new(initial_rtt: Duration) -> Self {
```

Comment:
this seems to not be needed anymore

Reply:
This is still needed by the `reset_congestion_state` function

Reply:
right, I missed that